bench_test.go

```diff
@@ -1,6 +1,7 @@
 package basictracer
 
 import (
+	"bytes"
```

Comment:
(this commit has the diffs from the simplified carriers that don't use `interface`s)